Makefile

```
@@ -0,0 +1,21 @@
+PREFIX?=/usr/local
+DESTDIR?=
+
+CC = gcc
```

This allows use of `scan-build` and other static analysis tooling.
```suggestion
CC ?= gcc
```

Consequently...
```sh
scan-build $(scan-build --help-checkers | xargs -L1 echo '--enable-checker' | xargs echo) make -B
```
...comes back clean!